pytest_fixtures.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+
+@pytest.fixture(scope='module')
+def db_url():
+    """ Database URL used in sqlalchemy.create_engine
+
+    Override this in your test to provide your desired url.
+    For valid urls see: http://docs.sqlalchemy.org/en/latest/core/engines.html
+    Defaults to SQLite memory database.
+    """
+    return 'sqlite://'
+
+
+@pytest.fixture(scope='module')
+def base():
+    error = """
+    Override this fixture in your tests to return
+    `sqlalchemy.ext.declarative.declarative_base` e.g.
+
+    from sqlalchemy.ext.declarative import declarative_base
+
+    class Base:
+        pass
+
+    Base = declarative_base(cls=Base)
+
+    @pytest.fixture(scope='module')
+    def base():
+        return Base
+    """
+    raise NotImplementedError(error)
```

Comment:
providing a custom `cls` is optional and would probably make a nice default here